explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -91,12 +91,18 @@ case class AladinCell(
       paConstraint  <- conf.posAngleConstraint
       angles        <-
         paConstraint.anglesToTestAt(configuration.siteFor, asterism.baseTracking, vizTime)
-    } yield angles
+    } yield {
+      // We sort the angles or we could end up in a loop where the angles are tested back and forth
+      // This is rare but can happen if each angle finds an equivalent guide star
+      given Order[Angle] = Angle.AngleOrder
+      angles.sorted
```

Comment:
`angles.sorted(using Angle.AngleOrder)` ?

Reply:
Oh didn't know that